docker/devbox/Dockerfile

```diff
@@ -9,7 +9,9 @@ FROM artifactory.wikia-inc.com/sus/php-wikia-base:06b60d8
 # these require autoconf package to be present
 
 RUN pecl install yaml-2.0.2 xdebug-2.6.0 \
-    && docker-php-ext-enable yaml xdebug
+    && docker-php-ext-enable yaml xdebug \
+    && pecl install apcu \
+    && docker-php-ext-enable apcu
```

Comment:
This will need to be installed on prod as well, I guess

Reply:
Good catch:
* 88fd14bf2e06969d5a03fcbc67e3d09a8bc5ccdf
* 51e6acc0b6f4670bfc9641ddfa85668a0a136b39